src/apps/bm_devkit/bmdk_common/app_main.cpp

```diff
@@ -395,6 +394,27 @@ static void defaultTask(void *parameters) {
   sensorConfig_t sensorConfig = {.sensorCheckIntervalS = sys_cfg_sensorsCheckIntervalS,
                                  .sensorsPollIntervalMs = sys_cfg_sensorsPollIntervalMs};
   sensorSamplerInit(&sensorConfig);
+#if APP == bm_loadcell
+  NAU7802 load_cell(&i2c1, NAU7802_ADDR);
+  LoadCellConfig_t load_cell_cfg = {
+      .calibration_factor = DEFAULT_CALIBRATION_FACTOR,
+      .zero_offset = DEFAULT_ZERO_OFFSET,
+  };
+  if (!debug_configuration_system.getConfig("loadCellCalibrationFactor",
+                                            strlen("loadCellCalibrationFactor"),
+                                            load_cell_cfg.calibration_factor)) {
+    debug_configuration_system.setConfig("loadCellCalibrationFactor",
+                                         strlen("loadCellCalibrationFactor"),
+                                         load_cell_cfg.calibration_factor);
+  }
+  if (!debug_configuration_system.getConfig("loadCellZeroOffset", strlen("loadCellZeroOffset"),
+                                            load_cell_cfg.zero_offset)) {
+    debug_configuration_system.setConfig("loadCellZeroOffset", strlen("loadCellZeroOffset"),
+                                         load_cell_cfg.zero_offset);
+  }
+  // Adds the load cell as a sensor for periodic sampling.
+  loadCellSamplerInit(&load_cell, load_cell_cfg);
+#endif
```

Comment:
Load cell is now initialized here rather than in user code, this will only occur if the app is `bm_loadcell`